src/routers/commentRouter.js

```diff
@@ -1,37 +1,12 @@
 import express from 'express';
-import threadRouter from './threadRouter.js';
+import * as commentController from '../controllers/commentController.js';
 
 const commentRouter = express.Router();
 
-// TODO:
-const createComment = async (req, res) => {
-  console.log('Server::createComment');
-};
-
-// TODO:
-const getComment = async (req, res) => {
-  console.log('Server::getComment');
-};
-
-// TODO:
-const updateComment = async (req, res) => {
-  console.log('Server::updateComment');
-};
-
-// TODO:
-const patchComment = async (req, res) => {
-  console.log('Server::patchComment');
-};
-
-// TODO:
-const deleteThread = async (req, res) => {
-  console.log('Server::deleteThread');
-};
-
-threadRouter.post('/', createComment);
-threadRouter.get('/:id', getComment);
-threadRouter.put('/:id', updateComment);
-threadRouter.patch('/:id', patchComment);
-threadRouter.delete('/:id', deleteThread);
+commentRouter.post('/', commentController.createComment);
+commentRouter.get('/:id', commentController.getComment);
```

Comment:
Also we need an API to get all comments of a given thread.

Reply:
ok, I will try add a getComments